Source/Core/HeadingPitchRoll.js

```diff
@@ -0,0 +1,145 @@
+/*global define*/
+define([
+    './defaultValue',
+    './defined',
+    './DeveloperError',
+    './freezeObject',
+    './Math',
+    './defineProperties'
+], function (
+        defaultValue,
+        defined,
+        DeveloperError,
+        freezeObject,
+        CesiumMath,
+        defineProperties) {
+    "use strict";
+
+    /**
+     * heading pitch roll type. Heading is the rotation about the
```

Comment:
Use complete sentences.  Maybe "A rotation expressed as a heading, pitch, and roll."